components/cli/cmd/cellery/list_Instances.go

```diff
@@ -25,15 +25,17 @@ import (
 )
 
 func newListInstancesCommand() *cobra.Command {
+	var verboseMode = false
 	cmd := &cobra.Command{
 		Use:     "instances",
 		Short:   "List all running cells",
 		Aliases: []string{"instance", "inst"},
 		Args:    cobra.NoArgs,
 		Run: func(cmd *cobra.Command, args []string) {
-			commands.RunListInstances()
+			commands.RunListInstances(verboseMode)
 		},
 		Example: "  cellery list instances",
 	}
+	cmd.Flags().BoolVarP(&verboseMode, "verbose", "v", false, "Run on verbose mode")
```

Comment:
Can't we use `cmd.PersistentFlags()`? It will propagate the flag to all subcommands.

Reply:
Updated.